src/main/java/com/eternalcode/parcellockers/configuration/implementation/ConfigItem.java

```diff
@@ -51,8 +90,24 @@ public ConfigItem setLore(List<String> lore) {
         return this;
     }
 
+    public ConfigItem addLore(String lore) {
+        this.lore.add(lore);
+        return this;
+    }
+
     public ConfigItem setGlow(boolean glow) {
         this.glow = glow;
         return this;
     }
+
+    @Override
+    public ConfigItem clone() {
+        try {
+            ConfigItem cloned = (ConfigItem) super.clone();
+            cloned.lore = new ArrayList<>(this.lore); // Deep copy of the list
+            return cloned;
+        } catch (CloneNotSupportedException e) {
```

Comment:
in many places of the code you improve the style of the code from the main branch, but here you have not changed it anymore?